tools/explorer/lib/tests/ontology_wrapper_test.py

```diff
@@ -0,0 +1,80 @@
+"""Testing module for ontology.py"""
+from absl.testing import absltest
+
+from yamlformat.validator import namespace_validator as nv
+
+from lib.ontology_wrapper import OntologyWrapper
+from lib.model import EntityTypeField
+from lib.model import StandardField
+
+from validate.universe_helper.config_universe import create_simplified_universe
+
+class OntologyTest(absltest.TestCase):
+
+  def setUp(self):
+    super(OntologyTest, self).setUp()
+    self.universe = create_simplified_universe()
+    nv.NamespaceValidator(self.universe.GetEntityTypeNamespaces())
+    self.ontology = OntologyWrapper(self.universe)
+
+  def testGetAllFieldsForTypeName(self):
+    expected_output = [
+        EntityTypeField('', 'manufacturer_label', True),
+        EntityTypeField('', 'model_label', True),
+        EntityTypeField('', 'exhaust_air_damper_command', False),
+        EntityTypeField('', 'exhaust_air_damper_status', False)
+    ]
+    expected_output_sorted = sorted(
+        expected_output,
+        key=lambda x: x.GetStandardFieldName(),
+        reverse=False
+    )
+
+    function_output = self.ontology.GetFieldsForTypeName(
+        'HVAC',
+        'DMP_EDM'
+    )
+    function_output_sorted = sorted(
+        function_output,
+        key=lambda x: x.GetStandardFieldName(),
+        reverse=False
+    )
+
+    self.assertEqual(function_output_sorted, expected_output_sorted)
+
+  def testGetRequiredFieldsForTypeName(self):
+    expected_output = [
+        EntityTypeField('', 'exhaust_air_damper_command', False),
+        EntityTypeField('', 'exhaust_air_damper_status', False)
+    ]
+    expected_output_sorted = sorted(
+        expected_output,
+        key=lambda x: x.GetStandardFieldName(),
+        reverse=False
+    )
+
+    function_output = self.ontology.GetFieldsForTypeName(
+        namespace='HVAC',
+        entity_type_name='DMP_EDM',
+        required_only=True
+    )
+    function_output_sorted = sorted(
```

Comment:
why not bake the sorting in the function itself?

Reply:
Good call. Done.

Reply:
I am still seeing sorted in the test here?